engine/gamesys/src/gamesys/scripts/script_physics.cpp

```diff
@@ -1398,6 +1398,14 @@ namespace dmGameSystem
         return 1;
     }
 
+    static void Physics_CheckIfShapeSizeValid(lua_State* L, float value)
+    {
+        if (value == 0.0f)
```

Comment:
I think you typically want to compare with an epsilon when comparing floats, there is no guarantee it is exactly 0.0f